IN-CORE/incore-services
# 188 redo adding owner item to dataset and delete by only owner

To test this PR:

1. Create any dataset, hazard, fragility curves, or anything. 
2. After the new dataset created, check if there is owner part in the metadata. It should be there and the owner should be your name
3. Try to delete the newly created dataset. However, test it with the different username. You can change the preffered_username in the header to someone else or just not the owner.
4. It should fail, if the preffered_username is not the same as the owner.
5. Use the owner as preffered_username, then it should be deleted
